ui/components/ui/icon-border/icon-border.scss

```diff
@@ -1,8 +1,9 @@
 .icon-border {
   border-radius: 50%;
-  border: 1px solid #f2f3f4;
-  background: #ececf0;
+  border: 1px solid $ui-1;
+  background: $ui-1;
   display: flex;
   justify-content: center;
   align-items: center;
+  overflow: hidden;
 }
```

Comment:
Checked instances of the `<SiteIcon>` component and visual changes should be minimal. There is a hex value above that isn't apart of our official color palette.

<img width="1011" alt="Screen Shot 2021-11-23 at 3 51 25 PM" src="https://user-images.githubusercontent.com/8112138/143147160-7b44f3a1-a011-4586-8844-551c3a435b33.png">
<img width="970" alt="Screen Shot 2021-11-23 at 3 52 20 PM" src="https://user-images.githubusercontent.com/8112138/143147163-9b1785b7-d162-403f-a379-09c1b0093bc7.png">
<img width="464" alt="Screen Shot 2021-11-23 at 3 53 28 PM" src="https://user-images.githubusercontent.com/8112138/143147164-16378887-f50e-4ffb-b020-deb5c9524104.png">

Reply:
Resolved in #12797